integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -45,22 +49,54 @@ def __init__(
         :param collection_name: the name of the collection to use in the database.
         :param embedding_function: the name of the embedding function to use to embed the query
         :param persist_path: where to store the database. If None, the database will be `in-memory`.
+        :param distance_function: The distance metric for the embedding space.
+            - `"l2"` computes the Euclidean (straight-line) distance between vectors,
+            where smaller scores indicate more similarity.
+            - `"cosine"` computes the cosine similarity between vectors,
+            with higher scores indicating greater similarity.
+            - `"ip"` stands for inner product, where higher scores indicate greater similarity between vectors.
+            **Note**: `distance_function` can only be set during the creation of a collection.
+            To change the distance metric of an existing collection, consider cloning the collection.
+            For more details, see the [documentation](https://cookbook.chromadb.dev/core/collections/).
```

Comment:
the link refers to unofficial docs

Reply:
I couldn't find an explanation in the official docs regarding the inability to reassign the distance metric. Hence the link to the cookbook. I can remove the link. WDYT?

Reply:
yes, let's remove it